webapps/ui/admin/client/scripts/pages/diagnostics.js

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership. Camunda licenses this file to you under the Apache License,
+ * Version 2.0; you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+var fs = require('fs');
+
+var template = fs.readFileSync(__dirname + '/diagnostics.html', 'utf8');
+
+var Controller = [
+  '$scope',
+  'camAPI',
+  'Notifications',
+  '$translate',
+  function($scope, camAPI, Notifications, $translate) {
+    var telemetryResource = camAPI.resource('telemetry');
+
+    telemetryResource.fetchData(function(err, res) {
+      if (err) {
+        const msg = err.message;
+        Notifications.addError({
+          status: $translate.instant('ERROR'),
+          message: $translate.instant('DIAGNOSTICS_FETCH_DATA_ERROR_MESSAGE', {
+            msg
+          })
+        });
```

Comment:
I pushed directly on this branch a small improvement to change the translation string from `TELEMETRY_FETCH_DATA_ERROR_MESSAGE` to `DIAGNOSTICS_FETCH_DATA_ERROR_MESSAGE`. Also, I passed the `err.message` directly into the `#instant()` function instead of string concatenation.